CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+# Contributing Guidelines
+
+Thanks for interest in contributing to MTF.
+
+The following is a set of guidelines for contributing to MTF. These are not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## How to contribute to MTF
+
+### Reporting Bugs
+
+Before creating bug reports, please check a [list of known issues](https://github.com/fedora-modularity/meta-test-family/issues) to see if the problem has already been reported.
+
+If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/fedora-modularity/meta-test-family/issues/new). Be sure to include a **descriptive title and clear description** and please include **as many details as possible** to help maintainers reproduce the issue and resolve it faster. If possible, add a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+### Suggesting Enhancements
+
+Enhancement suggestions are tracked as [GitHub issues](https://guides.github.com/features/issues/). When you are creating an enhancement issue, **use a clear and descriptive title** and **provide a clear description of the suggested enhancement** in as many details as possible.
+
+### Submitting changes
+
+To submitt changes, please send a [GitHub Pull Request](https://github.com/fedora-modularity/meta-test-family/pull/new/devel). Before submitting a PR, please read the [Styleguides](#styleguides) to know more about coding conventions used in this project. Also we will appreciate if you check your code with pylint and pyflakes. Always create a new branch for each pull request to avoid intertwingling different features or fixes on the same branch. Always do "git pull --rebase" and "git rebase" vs "git pull" or "git merge".
```

Comment:
Please mention last sentence on separate row. It would be more clear. I have missed it by reading whole paragraph.